src/assets/styles/layout.css

```diff
@@ -1,234 +0,0 @@
-article .hero > section,
-article .hero div > section,
-article .main > section,
-article .asides aside > section {
-  max-width: 1060px;
-  margin-right: auto;
-  margin-left: auto;
-  padding-right: 15px;
-  padding-left: 15px;
-}
-@media screen and (min-width: 768px) {
-  article .hero > section,
-  article .hero div > section,
-  article .main > section,
-  article .asides aside > section {
-    padding-right: 30px;
-    padding-left: 30px;
-  }
-}
-::selection {
-  background: #f59f51;
-  color: #fff;
-}
-main {
-  display: block;/* IE 11 gives auto/no height to main */
-  min-height: 200px;
-  background: #f0f0f0;
-  padding-top: 50px;
-  position: relative;
-}
-@media screen and (min-width: 768px) {
-  main {
-    min-height: 400px;
-    padding-top: 73px;
-  }
-}
-h1,
```

Comment:
moved to app.less https://github.com/PRX/styleguide.prx.org/pull/20/files#diff-478c8efeb2d9a5f992f435f5dcd2afb5R6